### ISSUE TITLE
LottieAnimationView setProgress() not working as expected

### ISSUE THREAD

**Issue by wingoku** [airbnb/lottie-android#225]
I have a fragment inside an activity. Upon orientation change, the following code is executed in the fragment running LottieAnimationView. The code is supposed to _resume the animation from the same progress at which the fragment was destroyed_ upon orientation change. However in onActivityCreated() after orientation change, the **animation restarts from the beginning**. How can I fix it?

## EDIT:

`LottieAnimationView.setProgress()` doesn't work anyway even if I run the code like this with **fixed orientation**. It just starts from the beginning no matter what.

```
LottieAnimationView.setProgress(0.5);
LottieAnimationView.playAnimation();
```

```
 @Override
    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
        super.onActivityCreated(savedInstanceState);
        if(savedInstanceState != null) {
            mAnimationView.setProgress(savedInstanceState.getFloat(Constants.KEY_LOTTIE_ANIMATION_PROGRESS));
            mAnimationView.resumeAnimation();
        }
        else {
            mAnimationView.playAnimation();
        }
    }

    @Override
    public void onSaveInstanceState(Bundle outState) {
        super.onSaveInstanceState(outState);
        outState.putFloat(Constants.KEY_LOTTIE_ANIMATION_PROGRESS, mAnimationView.getProgress());
    }
```

**Comment by bmc08gt** [comment]
#228 Seems to be working as expected after this change

**Comment by gpeal** [comment]
Fixed by #228
